### PR TITLE
Rhel 8 add infra check

### PR DESCRIPTION
**Pull request by jkonecny12**
Backport of https://github.com/rhinstaller/anaconda/pull/3429 https://github.com/rhinstaller/anaconda/pull/3466 and https://github.com/rhinstaller/anaconda/pull/3467.

~[Test run](https://github.com/jkonecny12/anaconda/runs/2885238403?check_suite_focus=true).~ New [test run](https://github.com/jkonecny12/anaconda/runs/2896319223?check_suite_focus=true).

### CODE REVIEW

**Comment by jkonecny12** [comment]
Hi @jstodola, asking for exception.

**Review by VladimirSlavik — APPROVED** [review]
Thank you!

**Review by poncovka — APPROVED** [review]

**Comment by jkonecny12** [comment]
This PR have to be extended by https://github.com/rhinstaller/anaconda/pull/3466 .

**Review by poncovka — APPROVED** [review]

File: .structure-config [hunk 1]
```diff
@@ -0,0 +1 @@
+INFRASTRUCTURE_FILES='.structure-config .packit.yml .pep8speaks.yml .codecov.yml .coveragerc .github/ dockerfile/ scripts/testing/ scripts/githooks/'
```
Comment (jstodola) [inline .structure-config:1]: `.coveragerc` is part of anaconda-33.16.5.2-1.el8.src.rpm - is it expected? If yes, then is should not be marked as an infrastructure file.

Comment (jkonecny12) [reply .structure-config:1]: I don't think we want to have `.coveragerc` in the tarball but I could be wrong. @poncovka do you know why it is in tarball?

Comment (poncovka) [reply .structure-config:1]: For historical reasons (https://github.com/rhinstaller/anaconda/commit/b9dcc94b5e114e9bd03b441342ccfa131d7a9298)? I think we can drop it.

Comment (jkonecny12) [reply .structure-config:1]: OK, I'll add new commit to drop the `.coveragerc` from tarball. Thanks!

**Comment by jkonecny12** [comment]
**UPDATED:**
* coveragerc was removed from the tarball

**Review by VladimirSlavik — APPROVED** [review]

**Review by jstodola — APPROVED** [review]
It looks good to me, thanks.

**Pull request merged**